src/logic/hooks/useDarkMode.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect } from 'react'
+import useCachedState from 'src/utils/storage/useCachedState'
+
+const useDarkMode = (): [boolean, (mode: boolean) => void] => {
+  const [darkMode = false, setDarkMode] = useCachedState<boolean>('darkMode')
```

Comment:
It should probably default to [user preference](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) to.

Reply:
Even if this is just an experimental feature?

Reply:
True, I didn't consider that. Perhaps not then.